app/Services/Database/AbstractDatabase.php

```
@@ -238,6 +240,15 @@ public function unlink(string $username, string $host): void
      */
     public function runBackup(BackupFile $backupFile): void
     {
+        $backupFile->database_engine = $this->service->name;
+
+        try {
+            $backupFile->database_version = BackupFile::normalizeVersion($this->version());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\BackupFile' in method 'runBackup'. (undefined)

(StaticAccess)